guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -43,13 +52,64 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 ----
 # subscription-manager repos --enable=rhel-7-server-extras-rpms
 ----
+ifdef::satellite[]
+.. Enable the `{RepoRHEL7Server}` repository:
++
+[options="nowrap" subs="quotes, attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL7Server}
+----
+.. Enable the `{RepoRHEL7ServerSatelliteServerProductVersion}` repository:
++
+[options="nowrap" subs="quotes, attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL7ServerSatelliteServerProductVersion}
+----
++
+In a disconnected environment, these repositories can come from a locally mounted ISO. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html-single/installing_satellite_server_in_a_disconnected_network_environment/index#configuring-the-base-operating-system-with-offline-repositories-in-rhel-7_satellite[Configuring the Base Operating System with Offline Repositories in RHEL 7] in _Installing Satellite Server in a Disconnected Network Environment_.
+ endif::[]
+endif::[]
 
 . Install required packages:
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 +
 ----
 # {package-install-project} leapp leapp-repository
 ----
+ifdef::satellite[]
++
+. For Leapp to perform the upgrade in a disconnected environment, the metadata should be downloaded and extracted manually, as described in https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+
+. Set up the following repositories to perform the upgrade in a disconnected environment:
+.. `/etc/yum.repos.d/rhel8.repo`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[BaseOS]
+name={RepoRHEL8BaseOS}
+baseurl=http://server.example.com/rhel8/BaseOS/
+[AppStream]
+name={RepoRHEL8AppStream}
+baseurl=http://server.example.com/rhel8/AppStream/
+----
++
+.. `/etc/yum.repos.d/satellite.repo:`
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={RepoRHEL8ServerSatelliteServerProductVersion}
+baseurl=http://server.example.com/sat6/Satellite/
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+baseurl=http://server.example.com/sat6/Maintenance/
```

Comment:
also use attributes if possible.

Reply:
@evgeni are there attributes for the base URLs?

Reply:
I don't think we do, no. And we didn't use any in `guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc` either.